docs/pages/guides/customizing-webpack.md

```diff
@@ -77,13 +77,12 @@ The reason it automatically includes the polyfill is because `react-native-web`
 
 ## Editing static files
 
-You can also use `expo customize:web` to generate the static project files: **index.html**, **serve.json**, **favicon.ico**, etc...
+You can also use `npx expo customize` to generate the static project files: **index.html**, **serve.json**, etc...
```

Comment:
```suggestion
You can also use `npx expo customize` to generate the static project files: **index.html**, **serve.json**, etc.
```

Unrelated to your changes but I think since we are updating this page, let's fix this typo 👍